src/test/scala/nl.knaw.dans.easy.authinfo/components/FileItemSpec.scala

```diff
@@ -52,7 +57,10 @@ class FileItemSpec extends TestSupportFixture {
          |  "owner":"someone",
          |  "dateAvailable":"1992-07-30",
          |  "accessibleTo":"KNOWN",
-         |  "visibleTo":"ANONYMOUS"
+         |  "visibleTo":"ANONYMOUS",
+         |  "licenseKey":"http://opensource.org/licenses/MIT",
+         |  "licenseTitle":"MIT.txt"
+         |
```

Comment:
not sure, but I suspect this newile in not necessary

```suggestion
```

Reply:
correct